test/unit/sql/index.test.js

```diff
@@ -17,14 +17,14 @@ suite(Support.getTestDialectTeaser('SQL'), function() {
 
       if (current.dialect.supports.schemas) {
         expectsql(sql.addIndexQuery('schema.table', ['column1', 'column2'], {}), {
-          default: 'CREATE INDEX [schema_table_column1_column2] ON [schema].[table] ([column1], [column2])'
+          default: 'CREATE INDEX [schema_table_column1_column2] ON [schema.table] ([column1], [column2])'
         });
 
         expectsql(sql.addIndexQuery(sql.quoteTable(sql.addSchema({
           schema: 'schema',
           tableName: 'table'
         })), ['column1', 'column2'], {}), {
-          default: 'CREATE INDEX [schema_table_column1_column2] ON [schema].[table] ([column1], [column2])'
+          default: 'CREATE INDEX [schema_table_column1_column2] ON [schema.table] ([column1], [column2])'
```

Comment:
I'll undo the change to this file in hopes that I can make a change in the query-generator.js that prevents other specs from needing to be changed.